src/layout/grid.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+interface GridAreaItemProps {
+  area: string;
+}
+export const GridAreaItem = styled.div<GridAreaItemProps>`
+  grid-area: ${p => p.area};
+`;
```

Comment:
Can you add in the comments to this file for structure also can this become a React component?